server/catgenome/src/test/java/com/epam/catgenome/util/TabixReaderTest.java

```diff
@@ -0,0 +1,173 @@
+package com.epam.catgenome.util;
+
+import com.epam.catgenome.util.feature.reader.TabixIteratorLineReader;
+import com.epam.catgenome.util.feature.reader.TabixReader;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringJUnit4ClassRunner.class)
```

Comment:
This is quite minor, but I think, you can get rid of Spring context here, since you don't need Dependency Injection. This will make this a pure Unit Test and it will be much faster. If you'll dcide to change it, don't forget to change the way you treat test resources to getClass().getClassLoader().getResource(fileName).getFile()